Lib/asyncio/tasks.py

```diff
@@ -107,11 +109,17 @@ def __del__(self):
             if self._source_traceback:
                 context['source_traceback'] = self._source_traceback
             self._loop.call_exception_handler(context)
-        futures.Future.__del__(self)
+        super().__del__()
 
     def _repr_info(self):
         return base_tasks._task_repr_info(self)
 
+    def set_result(self, result):
+        raise RuntimeError('Tasks do not support set_result operation')
```

Comment:
`Task does not support ...` sounds slightly better to me but I don't insist.

Reply:
done